### ISSUE TITLE
Internal structure is appended the beginning of the `before`

### ISSUE THREAD

**Issue by raczben** [raczben/wexpect#6]
## Report a bug

Some unwanted data is appended the beginning of the `before` field.

### Reproduce

Run the hello_wexpect examlpe
`python examples\hello_wexpect.py`

### Output

```
D:\bt\wexpect
yINPUT_RECORD(EventType=1) (KEY_EVENT), PyINPUT_RECORD(EventType=1) (KEY_EVENT)]
{'Size': PyCOORDType(X=80,Y=16000), 'CursorPosition': PyCOORDType(X=14,Y=3), 'Attributes': 7, 'Window': PySMALL_RECTType(Left=0,Top=0,Right=79,Bottom=24), 'MaximumWindowSize': PyCOORDType(X=80,Y=64)}
lsPyINPUT_RECORD(EventType=1) (KEY_EVENT)]
{'Size': PyCOORDType(X=80,Y=16000), 'CursorPosition': PyCOORDType(X=2,Y=8), 'Attributes': 7, 'Window': PySMALL_RECTType(Left=0,Top=0,Right=79,Bottom=24), 'MaximumWindowSize': PyCOORDType(X=80,Y=64)}

__pycache__  coverage.xml  LICENSE     setup.cfgwexpect.egg-info
AUTHORS      examples   README.md     setup.pywexpect.py
ChangeLog    htmlcov   requirements.txt  testswexpect.pyc

D:\bt\wexpect
```

### Expected 

The following is unwanted
```
yINPUT_RECORD(EventType=1) (KEY_EVENT), PyINPUT_RECORD(EventType=1) (KEY_EVENT)]
{'Size': PyCOORDType(X=80,Y=16000), 'CursorPosition': PyCOORDType(X=14,Y=3), 'Attributes': 7, 'Window': PySMALL_RECTType(Left=0,Top=0,Right=79,Bottom=24), 'MaximumWindowSize': PyCOORDType(X=80,Y=64)}
lsPyINPUT_RECORD(EventType=1) (KEY_EVENT)]
{'Size': PyCOORDType(X=80,Y=16000), 'CursorPosition': PyCOORDType(X=2,Y=8), 'Attributes': 7, 'Window': PySMALL_RECTType(Left=0,Top=0,Right=79,Bottom=24), 'MaximumWindowSize': PyCOORDType(X=80,Y=64)}
```

**Comment by raczben** [comment]
The bug comes with the 2 to 3 conversion. Python-27 with wexpect 2.3.2 runs perfectly.

**Comment by raczben** [comment]
The rubbish text is on the child terminal. Pause and view the child terminal using the `interact()` method.

**Comment by raczben** [comment]
Isuue fixed in [release 2.3.6](https://pypi.org/project/wexpect/2.3.6/)
